packages/modules/redis/src/redis-container.test.ts

```diff
@@ -102,6 +102,20 @@ describe("RedisContainer", { timeout: 240_000 }, () => {
   });
   // }
 
+  // startWithRedisStack {
+  it("should start with redis-stack-server and json module", async () => {
+    const container = await new RedisContainer("redis/redis-stack-server:latest").withPassword("testPassword").start();
```

Comment:
Let's pin a version instead of using `latest`, otherwise this test will break over time